app/Http/Controllers/GroupController.php

```diff
@@ -1108,13 +1108,13 @@ public function volunteersNearby($groupid)
         }
 
         //Event tabs
-        $upcoming_events = Party::upcomingEvents()
-            ->where('events.group', $group->idgroups)
+        $upcoming_events = Party::futureForUser()
+            ->forGroup($group->idgroups)
```

Comment:
This seems like an interesting combo - future for user for a specific group - is that right?